<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtQueryTesterWindow</class>
 <widget class="QMainWindow" name="QtQueryTesterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1879</width>
    <height>1086</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtQueryTesterWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>XQuery:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Result:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>XML:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="textQuery"/>
    </item>
    <item row="3" column="0">
     <widget class="QPlainTextEdit" name="textXml"/>
    </item>
    <item row="1" column="1" rowspan="3">
     <widget class="QPlainTextEdit" name="textResult"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1879</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
